elements/pfe-band/src/pfe-band.scss

```diff
@@ -103,246 +179,206 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
-            "header header"
-            "aside body"
-            "footer footer";
-        }
-        :host([has_header][has_footer][has_aside][aside-desktop="left"][aside-height="full"]) & {
-          grid-template-areas:
-            "aside header"
-            "aside body"
-            "aside footer";
-        }
+      :host([has_aside][aside-desktop="left"]) & {
+        grid-template-rows: auto;
+        grid-template-columns: pfe-local(layout, $fallback: pfe-local(Width__aside--lg) 1fr);
       }
     }
   }
+}
 
-  // For each section in the band layout
-  @each $section in (header, body, aside, footer) {
-    &__#{$section} { // i.e., pfe-band__header
-      margin-bottom: pfe-local(gutter--vertical);
-      // Apply the standard grid
-      @supports (display: grid) {
+.pfe-band__container,
+.pfe-band__wrapper {
+    // Fallback margins if grid is not used
+    margin-bottom: calc(-1 * #{pfe-local(gutter--vertical)});
+    margin-right: calc(-1 * #{pfe-local(gutter--vertical)});
+    @supports (display: grid) {
+      margin: 0 auto;
+    }
+}
+
+.pfe-band__aside {
+  flex-grow: 1;
+  align-self: flex-start;
+}
+
+@each $section in (header, body, footer, aside) {
+  .pfe-band__#{$section} {
+    // Fallback margins if grid is not used
+    margin-bottom: pfe-local(gutter--vertical, $region: $section);
+    margin-right: pfe-local(gutter--vertical, $region: $section);
+
+    // Apply the grid layout
+    @supports (display: grid) {
+      // Apply the section as a named grid-area
+      grid-area: $section;
+      margin-bottom: 0;
+      margin-right: 0;
+
+      :host([use-grid]) & {
         display: grid;
-        row-gap: pfe-local(gutter--vertical);
-        column-gap: pfe-local(gutter--horizontal);
-        margin-bottom: 0;
-        // Apply the section as a named grid-area
-        grid-area: $section;
-        // Allow users to hook into the layout with custom columns
-        grid-template-columns: pfe-local(layout, $region: #{$section});
-        grid-template-rows:    auto;
+        grid-template-rows: auto;
+        grid-template-columns: pfe-local(layout, $region: $section);
+
+        grid-row-gap: pfe-local(gutter--vertical, $region: $section);
+        row-gap:      pfe-local(gutter--vertical, $region: $section);
+        grid-column-gap:  pfe-local(gutter--horizontal, $region: $section);
+        column-gap:       pfe-local(gutter--horizontal, $region: $section);
       }
     }
   }
+}
 
-  &__aside {
-    align-self: flex-start;
+// For each section in the band layout,
+// define their internal behavior for slotted elements
+@include browser-query(ie11) {
+  .pfe-band__container {
+    @include pfe-clearfix;
   }
 
-  @include browser-query(ie11) {
-    &__body {
-        width: 60%;
-        float: left;
+  .pfe-band__header,
+  .pfe-band__body,
+  .pfe-band__footer {
+    float: left;
+
+    :host([aside-mobile="top"]) & {
+      float: right;
+    }
+  
+    :host(:not([aside-height="full"])) & {
```

Comment:
It seems that this selector may not be working correctly? It's setting the width of the header & footer regions to 60% too. 
 
patternflyelements.com:
![image](https://user-images.githubusercontent.com/456863/100806235-19a6be80-33fe-11eb-9f2b-3809657c69bb.png)

this branch: 
![image](https://user-images.githubusercontent.com/456863/100806268-262b1700-33fe-11eb-8fcf-dd5f0fe2b0f9.png)

Reply:
Can you share the screen-size you were at?  It could be a case of needing a breakpoint too 🤔